### DISCUSSION TITLE
Impinging jet heat transfer correlation in FDS

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13838] [Q&A]
@rmdermo and @johodges , Thank you for implementing  the heat transfer correlations that is more  appropriate for jet impingement applications inside FDS. I have a query to check with you both, that if the model can  be used for high speed jets ca. 150 to 200 m/s at temperature range  -70 to -50 C .  The Prandtl number for the species will be around 0.8 in the gas phase for this temperature range during the  impact and Reynolds number ca. 10^7.

**Comment by johodges** [comment]
The correlation may work in that configuration, but those velocities are outside the range for FDS and beyond anything we have compared with to date. The user guide states a limit of Mach number < 0.3 which is ~100 m/s. I haven't seen anyone compare the FDS predictions outside that range with a compressible solver to see how much of an impact there is, but Randy may be aware of a study looking into it.

**Reply by Muthu-V** [reply]
Thank  you for your insights @rmcdermo and @johodges . Yes the flow regime is subsonic mach > 0.3 and well below 0.7. The jet flow properties are set at expanded conditions for atmospheric pressure 1 bar  where there is no shock regimes .  In reality once the jet is impinged at nearby obstacles , the flow regime is largely governed by the incompressible flow. Particularly when the flow of interest is to check the consequences of large jet releases after impingement at incompressible flow regimes.  In the Loughborough jet fire experiment validation test case,  the gas phase mach number can reach up to 0.74 in some cells near the release. The code is stable and mimics the subsonic part with incompressible flow assumption well.   Any thoughts on that. 
![image](https://github.com/user-attachments/assets/cd35d411-b659-4895-b672-158f84932827)
![jet1_1000](https://github.com/user-attachments/assets/c4894299-38e1-425d-ac66-2f6187fe1651)

**Comment by rmcdermo** [comment]
I agree that if you have the mass loss rate correct at the jet, then most of the domain is well within low-Mach validity.  Whether or not the IMPINGING JET model is going to do the trick for you in terms of predicting heat transfer to that cylinder, that is not something I can guarantee.  I would so some sanity checks with cylinder correlations.
